libs/sdk/src/common/push-protocol.ts

```diff
@@ -133,6 +152,49 @@ class PushProtocol {
     return inboxNotifications;
   }
 
+  /**
+   * Gets all unread notifications
+   * @returns {Promise<PushOrgNotification[]>}
+   */
+  async fetchUnreadNotifications() {
+    const notifications = await this.getNotifications(1, 100); // don't show more than 100 urnead notifications
+    const latestSid = localStorage.getItem(this.lastestSeenSidKey);
+    const latestSidNumber = latestSid ? parseInt(latestSid) : null;
+
+    if (latestSidNumber && !isNaN(latestSidNumber)) {
+      // Filter notifications that have a sid greater than the last seen sid
+      this._unreadNotifications = notifications.filter(
+        notification => Number(notification.sid) > latestSidNumber,
```

Comment:
Why did you use `Number` here instead of `parseInt`?

Reply:
changing to parseInt.